mitmproxy/addons/dumper.py

```diff
@@ -225,6 +226,8 @@ def echo_flow(self, f):
             self._echo_response_line(f)
             if ctx.options.flow_detail >= 2:
                 self._echo_headers(f.response.headers)
+                if f.response.trailers is not None and isinstance(f.response.trailers, http.Headers):
+                    self._echo_headers(f.response.trailers)
```

Comment:
Do you have a sample output how this will look like?
Is there a clear indication that these are trailers? Maybe add a divider line or prefix?

Reply:
I suppose logically the trailers should be printed after the message body (with `flow_detail >= 3` below)?